docs/sources/api/_index.md

```diff
@@ -560,337 +584,131 @@ $ curl -v -H "Content-Type: application/json" -XPOST -s "http://localhost:3100/l
-- `end`: The end time for the query as a nanosecond Unix epoch. Defaults to now.
+`/metrics` returns exposed Prometheus metrics. See
+[Observing Loki](../operations/observability/)
+for a list of exported metrics.
 
-In microservices mode, `/api/prom/label/<name>/values` is exposed by the querier.
+In microservices mode, the `/metrics` endpoint is exposed by all components.
 
-Response:
+## List current configuration
 
 ```
-{
-  "values": [
-    <label value>,
-    ...
-  ]
-}
+GET /config
 ```
 
-### Examples
+`/config` exposes the current configuration. The optional `mode` query parameter can be used to
+modify the output. If it has the value `diff` only the differences between the default configuration
+and the current are returned. A value of `defaults` returns the default configuration.
 
-```bash
-$ curl -G -s  "http://localhost:3100/api/prom/label/foo/values" | jq
-{
-  "values": [
-    "cat",
-    "dog",
-    "axolotl"
-  ]
-}
-```
+In microservices mode, the `/config` endpoint is exposed by all components.
 
-## `POST /api/prom/push`
+## List running services
 
-> **WARNING**: `/api/prom/push` is DEPRECATED; use `/loki/api/v1/push`
-> instead.
+```
+GET /services
+```
 
-`/api/prom/push` is the endpoint used to send log entries to Loki. The default
-behavior is for the POST body to be a snappy-compressed protobuf message:
+`/services` returns a list of all running services and their current states.
 
-- [Protobuf definition](https://github.com/grafana/loki/tree/master/pkg/logproto/logproto.proto)
-- [Go client library](https://github.com/grafana/loki/tree/master/pkg/promtail/client/client.go)
+Services can have the following states:
 
-Alternatively, if the `Content-Type` header is set to `application/json`, a
-JSON post body can be sent in the following format:
+- **New**: Service is new, not running yet (initial state)
+- **Starting**: Service is starting; if starting succeeds, service enters **Running** state
+- **Running**: Service is fully running now; when service stops running, it enters **Stopping** state
+- **Stopping**: Service is shutting down
+- **Terminated**: Service has stopped successfully (terminal state)
+- **Failed**: Service has failed in **Starting**, **Running** or **Stopping** state (terminal state)
+
+## List build information
 
 ```
-{
-  "streams": [
-    {
-      "labels": "<LogQL label key-value pairs>",
-      "entries": [
-        {
-          "ts": "<RFC3339Nano string>",
-          "line": "<log line>"
-        }
-      ]
-    }
-  ]
-}
+GET /loki/api/v1/status/buildinfo
 ```
 
-Loki can be configured to [accept out-of-order writes](../configuration/#accept-out-of-order-writes).
+`/loki/api/v1/status/buildinfo` exposes the build information in a JSON object. The fields are `version`, `revision`, `branch`, `buildDate`, `buildUser`, and `goVersion`.
 
-In microservices mode, `/api/prom/push` is exposed by the distributor.
+## Series
```

Comment:
Feel free to ignore if you think it's a bad idea: Since most other headlines start with a verb, what about doing the same here?

```suggestion
## List matching series
```